assets/js/ga.js

```diff
@@ -0,0 +1,6 @@
+(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
+ga('create', 'UA-49796218-32', 'auto');
```

Comment:
noscript, privacy badger, etc., even a "defer all scripts" addon can make this fail. Always a good idea to wrap this in `if (typeof ga === "function") { ... }` because this code should not lead to an error in the console in production.